contracts/exchange/TokenPriceProvider.sol

```diff
@@ -14,19 +15,17 @@ contract TokenPriceProvider {
 
     mapping(address => uint256) public cachedPrices;
 
-    function syncPrice(ERC20 token) public {
-        uint256 expectedRate;
-        (expectedRate,) = kyberNetwork().getExpectedRate(token, ERC20(ETH_TOKEN_ADDRESS), 10000);
-        cachedPrices[address(token)] = expectedRate;
+    function setPrice(ERC20 _token, uint256 _price) external onlyManager {
+        cachedPrices[address(_token)] = _price;
     }
```

Comment:
We should add a method to set a list of token prices in one call.

Reply:
Done. Sorry, I had missed that comment.